dpdispatcher/__init__.py

```diff
@@ -8,7 +8,8 @@
 dlog = logging.getLogger(__name__)
 dlog.setLevel(logging.INFO)
 # dlogf = logging.FileHandler(os.getcwd()+os.sep+SHORT_CMD+'.log')
-dlogf = logging.FileHandler('./'+os.sep+SHORT_CMD+'.log')
+# dlogf = logging.FileHandler('./'+os.sep+SHORT_CMD+'.log')
+dlogf = logging.FileHandler(os.path.join(os.environ['HOME'], SHORT_CMD+'.log'))
```

Comment:
`os.path.expanduser('~')`

https://docs.python.org/3/library/os.path.html#os.path.expanduser

Reply:
But why did you change `dlogf`?

Reply:
Somestime, dpdispatcher may be used by other software (for me, airflow)
And './' may means '/' , and we can't write things there.